src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,125 @@
+package frc.team2412.robot.subsystems;
+
+import static frc.team2412.robot.Hardware.*;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkFlex;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import java.util.Map;
+
+public class IntakeSubsystem extends SubsystemBase {
+	// Constants
+	public static final double INTAKE_IN_SPEED = 0.3;
+	public static final double INTAKE_OUT_SPEED = -0.7;
+
+	public static final double INDEX_IN_SPEED = 0.3;
+	public static final double INDEX_OUT_SPEED = -0.3;
+
+	public static final double FEEDER_IN_SPEED = 0.3;
+	public static final double FEEDER_OUT_SPEED = -0.3;
+
+	// Motors
+	private final CANSparkMax intakeMotorFront;
+	private final CANSparkMax intakeMotorBack;
+	private final CANSparkMax intakeMotorLeft;
+	private final CANSparkMax intakeMotorRight;
+
+	private final CANSparkFlex indexMotor;
+
+	private final CANSparkFlex feederMotor;
+
+	// Shuffleboard
+	private final GenericEntry setIntakeSpeedEntry =
+			Shuffleboard.getTab("Intake")
+					.addPersistent("Intake speed - ", INTAKE_IN_SPEED)
```

Comment:
Should be "Intake in speed", as that's what being set.

Reply:
Best to also rename the variable names to add `In` as well. For this that'd be setIntakeInSpeedEntry